backend/src/Altinn.AccessManagement.UI/Altinn.AccessManagement.UI/Controllers/LookupController.cs

```diff
@@ -23,20 +23,21 @@ public class LookupController : ControllerBase
         /// <param name="lookupService">service implementation for lookups</param>
         public LookupController(
             ILogger<LookupController> logger,
-            ILookupService lookupService)
+            ILookupService lookupService,
+            IHttpContextAccessor httpContextAccessor)
         {
             _logger = logger;
             _lookupService = lookupService;
+            _httpContextAccessor = httpContextAccessor;
```

Comment:
Adding a dependency but it's not being used?

Reply:
Reverted changes in file